ansible_collections/arista/cvp/molecule/cv_task_v3/converge.yml

```diff
@@ -0,0 +1,56 @@
+---
+- name: Test cv_task_v3
+  hosts: CloudVision
+  connection: local
+  gather_facts: false
+  collections:
+    - arista.cvp
+
+  vars:
+    CVP_DEVICES_MOVE_DEVICE_EXECUTE:
+      - fqdn: leaf1
+        parentContainerName: "Spine"
+
+    CVP_DEVICES_MOVE_DEVICE_CANCEL:
+      - fqdn: leaf2
+        parentContainerName: "Spine"
+
+  tasks:
+    # Running move_device to create task list
+    - name: Run CV_DEVICE_V3 To Move Device For Execute Task
+      arista.cvp.cv_device_v3:
+          devices: '{{CVP_DEVICES_MOVE_DEVICE_EXECUTE}}'
+          state: present
+      register: MOVE_DEVICE_DATA_EXECUTE
+
+    - name: Execute Task
+      arista.cvp.cv_task_v3:
+        tasks:
+          - "{{MOVE_DEVICE_DATA_EXECUTE.devices_moved.taskIds[0]}}"
+      register: EXECUTE_TASK_DATA
+
+    - name: Testing Execute Task
```

Comment:
Can we get the task facts from CVP and check if they got executed? and same for cancelled example

Reply:
Will complete this once PR #632 gets merged

Reply:
Fixed!